### A Tufts University Research Technology Workshop

## Description
This course covers the basics of Next Generation Sequencing Bioinformatics, with the goal of 
variant calling and interpretation.
The material is designed to be a 3 hour workshop and is run on [Tufts High Performance Compute (HPC) Cluster](https://access.tufts.edu/research-cluster-account).
For support with this workshop or the HPC cluster, email TTS Research Technology [tts-research@tufts.edu](mailto:ltts-research@tufts.edu)

## Goals
- Writing and running bash scripts
- Intro to several bioinformatics tools: BWA, Samtools, Picard, GATK, VEP
- Variant Calling, Annotation and Interpretation

<img src="img/workflow.png" width="200">

## Prerequisites

### Computational skills needed
- Introduction to [Linux](https://tufts.box.com/s/x9aflewr2qw59pcbgcghbo9muykbi4ju)
- Introduction to [HPC](https://tufts.box.com/s/yubnzxnpih14hd80mbfxqrkdri8s2nws)

### Materials Needed
- Chrome web browser
- Account on [Tufts HPC](https://access.tufts.edu/research-cluster-account)
- [VPN](https://access.tufts.edu/vpn) if accessing the HPC from off campus

## Schedule
- [Introduction](slides/intro_to_ngs_bioinformatics_into_13May20.pdf)
- [Setup](lessons/01_Setup.md)
- [Quality Control](lessons/02_Quality_Control.md)
- [Alignment](lessons/03_Alignment.md)
- [Alignment Cleanup](lessons/04_Alignment_Cleanup.md)
- [Variant Calling](lessons/05_Variant_Calling.md)
- [Annotation and Interpretation](lessons/06_Variant_Annotation.md)


## Acknowledgement
Much of this workshop has been adapted from the [HBC Training In Depth NGS Analysis Course](https://github.com/hbctraining/In-depth-NGS-Data-Analysis-Course)
